src/lib.rs

```diff
@@ -1,17 +1,46 @@
-mod connect;
-mod sc;
-mod replica;
-mod metadata;
 mod admin;
+mod connect;
+mod consumer;
+mod producer;
+mod fluvio;
 
-
-use shared::SharedScClient;
+use shared::SharedFluvio;
 
 mod shared {
 
     use std::sync::Arc;
-    use flv_client::ScClient;
+    use fluvio::Fluvio;
     use flv_future_aio::sync::RwLock;
 
-    pub type SharedScClient = Arc<RwLock<ScClient>>;
-}
\ No newline at end of file
+    pub type SharedFluvio = Arc<RwLock<Fluvio>>;
```

Comment:
I think Fluvio should be sharable, might need to put clone derive

Reply:
Arc is clone by default

Reply:
@nicholastmosher create issue on Fluvio to make it sharable on next release,  node-client can use it then.   There is no need for other lang client library to do that